tdrs-backend/docs/remote-development.md

```diff
@@ -0,0 +1,68 @@
+# Remote Development
```

Comment:
Thank you for following up on this! This is excellent but could you move it under "Technical Documentation" directory?